docs/getting-started-tutorial/2-the-lift-menu-system.adoc

```diff
@@ -0,0 +1,33 @@
+:idprefix:
+:idseparator: -
+:toc: right
+:toclevels: 2
+
+# The Lift Menu System
+
+Another distinguishing characteristic of Lift is that it is *secure by
+default*. Amongst other things, this means that you can't access a file in your
+`src/main/webapp` directory through your application unless you explicitly
+define that it's meant to be accessed. You define this using Lift's menu
```

Comment:
This isn't entirely true. If a `SiteMap` isn't defined at all, then we default to permitting access to all views under the webapp directory that aren't stored in a folder starting with `_`. Unless, of course, that's changed in a recent commit and I missed it. =)

Reply:
! Nice catch. Rephrased a bit to take that into account.